src/components/court-order-poa/package.json

```diff
@@ -1,11 +1,11 @@
 {
   "name": "@bcrs-shared-components/court-order-poa",
-  "version": "2.1.3",
+  "version": "2.1.4",
   "publishConfig": {
     "access": "public"
   },
   "dependencies": {
-    "@bcrs-shared-components/interfaces": "^1.0.46",
+    "@bcrs-shared-components/interfaces": "^1.0.47",
```

Comment:
This file shouldn't be updated. This, too, is done by step 14.

(The only time this file should be updated is if you add/remove a package dependency.)